.circleci/config.yml

```diff
@@ -19,6 +19,14 @@ node_unit_tests: &node_unit_tests
     - run:
         name: Create Checksum
         command: sh .circleci/checksum.sh /tmp/checksums.txt
+    - run:
+        name: Setup environment variables
+        command: |
+          echo "export CIRCLE_NODE_VERSION=\$(node --version | grep -oE 'v[0-9]+')" >> $BASH_ENV
```

Comment:
I think we should Log out major and minor version. What do you think about removing `grep`?

Reply:
Is it possible do something like this => `echo $(node --version)`?

Reply:
I agree, I'll update to log out the node version.

For some context, I'm trying to recreate something similar to [`$TRAVIS_NODE_VERSION`](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions-using-nvmrc), which only holds the major version. The grep is just changing it from `v10.16.1` --> `v10` via regex. This way, we don't need to go back and change the codecov ver comparison for each minor/patch node.js update.